Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/Module.ixx

```diff
@@ -0,0 +1,3 @@
+export module Module;
+
+export void MyFunc();
```

Comment:
Why is this file needed?

Reply:
Not sure... Seems to be generated but I removed it.